dockers/discoverer/k8s/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN GO_VERSION="$(cat GO_VERSION)" \
     && CGO_ENABLED=0 \
     && GO111MODULE=on \
     go build \
+    -a -tags netgo \
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg?style=flat-square)
could you format the order of the newly added options like agent-ngt?
https://github.com/vdaas/vald/blob/aee9fd4b52acf62ceab5debc1b06c79037dbea99/dockers/agent/ngt/Dockerfile#L72-L73